include/boost/beast/core/detail/lean_tuple.hpp

```diff
@@ -0,0 +1,86 @@
+//
+// Copyright (c) 2018 Damian Jarek (damian dot jarek93 at gmail dot com)
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+// Official repository: https://github.com/boostorg/beast
+//
+
+
+#ifndef BOOST_BEAST_DETAIL_LEAN_TUPLE_HPP
```

Comment:
Shouldn't this be `BOOST_BEAST_CORE_DETAIL_LEAN_TUPLE_HPP`?

Reply:
I pretend like the "core" directory doesn't actually exist. Because it shouldn't exist. The actual include file should be `<beast/detail/lean_tuple.hpp>`. Unfortunately if I were to do that then the Visual Studio project would have a really long, annoying list of header files that I could not collapse and still see other virtual folders.